homeassistant/components/sensor/statistics.py

```diff
@@ -84,13 +85,13 @@ def __init__(self, hass, entity_id, name, sampling_size, max_age):
         self._max_age = max_age
         self._unit_of_measurement = None
         self.states = deque(maxlen=self._sampling_size)
-        if self._max_age is not None:
-            self.ages = deque(maxlen=self._sampling_size)
+        self.ages = deque(maxlen=self._sampling_size)
 
-        self.median = self.mean = self.variance = self.stdev = 0
-        self.min = self.max = self.total = self.count = 0
-        self.average_change = self.change = 0
-        self.max_age = self.min_age = 0
+        self.count = 0
+        self.mean = self.median = self.stdev = self.variance = STATE_UNKNOWN
```

Comment:
Should be initialized as `None` and not `STATE_UNKNOWN`. Home Assistant is handling it.

Reply:
done